### PR TITLE
[Feature panel] Show inline message when feature selected with no observations

### LINKED ISSUE #477: [Feature panel] Show message when selected feature has no observations
Center text in feature panel:

> No observations found for the selected feature.

### PR DESCRIPTION
**Pull request by gauravchalana1**
Hi @gino-m , PFB screenshot related to #477 ; Text added, please suggest for text customization. Thank you :) 

Screen Shot 2021-05-02 at 12 43 53 PM

Fixes #477.

### CODE REVIEW

File: web-ng/src/app/components/feature-panel/feature-panel.component.scss [hunk 1]
```diff
@@ -14,31 +14,39 @@
...
+  margin: 16px;
 }
 .observation .field-label {
-    font-size: 12px;
-    line-height: 16px;
-    color: #70757A;
-    margin-bottom: 3px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #70757a;
```
Comment (gino-m) [inline web-ng/src/app/components/feature-panel/feature-panel.component.scss:25]: Please use Angular themes to retrieve colors defined in https://github.com/google/ground-platform/blob/master/web-ng/src/app/shared/styles/_variables.scss. 

Examples:
https://github.com/google/ground-platform/search?q=mat-color

**Review by gino-m — CHANGES_REQUESTED** [review]
Thanks for this, very cool. One small suggestion, otherwise LGTM!

**Comment by gino-m** [comment]
@gauravchalana1 Gentle ping!

**Comment by gauravchalana1** [comment]
Hi @gino-m , PFB screenshot and changes as suggested. Tyty :) 

Screen Shot 2021-05-24 at 10 20 04 PM

**Review by gino-m — DISMISSED** [review]
🚀  Rockin'!! Thanks!

**Comment by gino-m** [comment]
@gauravchalana1 Could you please resolve the merge conflicts when you get a chance?

**Comment by gauravchalana1** [comment]
Hi @gino-m , please re-review the PR. conflicts have been resolved. Tyty :)

**Comment by gino-m** [comment]
Great work, thanks Gaurav!

**Pull request merged**